python/heterocl/devices.py

```diff
@@ -189,8 +207,11 @@ class FPGA(Device):
     def __init__(self, vendor, model, **kwargs):
         if vendor not in ["xilinx", "intel"]: 
             raise DeviceError(vendor + " not supported yet")
-        assert "fpga_" + model in model_table[vendor], \
-            model + " not supported yet"
+        if model is not None:
+            assert "fpga_" + model in model_table[vendor], \
+                "{} not supported yet".format(model)
+        else:
+            model = model_table[vendor][0]
```

Comment:
Similar comment to the above. It's fine we have repeated entries in different devices.